cmd/init.go

```diff
@@ -39,10 +39,10 @@ func (c *InitCommand) Run(args []string) int {
 		c.UI.Info("virtualenv not found")
 		s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
 		s.Suffix = " Installing virtualenv..."
-		s.FinalMSG = "✓ virtualenv installed\n"
 		s.Start()
 		c.Trellis.Virtualenv.Install()
 		s.Stop()
+		c.UI.Info(color.GreenString("✓ virtualenv installed"))
```

Comment:
Oh interesting... I guess `FinalMSG` is kind of useless in this case so we can just log as normal after 👍